build.sh

```diff
@@ -0,0 +1,3 @@
+gem install rubygems-bundler
```

Comment:
@orta `build.sh` is one of the ways you can tell Surf how to build your project. I haven't added Ruby support yet (you can see the available platforms at https://github.com/surf-build/surf/tree/master/src/build-discoverers), so I had to add the explicit script